### ISSUE TITLE
Enlarged icons in feed article

### ISSUE THREAD

**Issue by jmac-atgit** [Ranchero-Software/NetNewsWire#4371]
On a regular basis, I see hugely enlarged icons in my article window from one of the feeds I follow.
Nearly all of the articles from
Homepage: https://reactormag.com/
Feed: https://reactormag.com/feed/
show this issue when an article is viewed in NetNewsWire 6.1.4, macOS Sonoma 14.7.

I've also seen this issue in one article in one other feed:
Homepage: https://www.privacyguides.org/articles/
Feed: https://blog.privacyguides.org/feed_rss_created.xml
The enlarged icons are seen in this article: https://www.privacyguides.org/articles/2024/09/18/onion-browser-review/ which is dated Sep 17, 2024 in NetNewsWire.

**Comment by brentsimmons** [comment]
Looks both of these feeds are using SVG drawing declarations instead of images.

**Comment by brentsimmons** [comment]
Is this still happening? If so, which version of NetNewsWire and which OS?

**Comment by jmac-atgit** [comment]
Yes, I'm still seeing the large graphics. Same feed: https://reactormag.com/feed/

macOS 26.5.2 / Mac Studio with M1 Max / NetNewsWire 7.1.1.

Actually, I did find a workaround of setting "Always Use Reader View" for that feed. This suppresses the large icons. The disadvantage of using Reader View is that the pages render more slowly, and, on occasion, there is a considerable delay is displaying the page.

**Comment by brentsimmons** [comment]
Fixed in 7.1.3.

**Comment by jmac-atgit** [comment]
I just downloaded 7.1.3 and am very happy to see the fix for the SVG issue. The feed articles look fine now.

Many thanks!
